carma_wm/include/carma_wm/WorldModel.h

```diff
@@ -140,6 +140,16 @@ class WorldModel
    */
   virtual LaneletRouteConstPtr getRoute() const = 0;
 
+    /*! \brief Get trackpos of the end of route point relative to the route
+   *
+   * \return Trackpos 
+   */
+  virtual TrackPos getRouteEndTrackPos() const = 0;
+
+  /*! \brief Set endpoint of the route
+   */
+  virtual void setRouteEndPoint(const lanelet::BasicPoint3d& end_point) = 0;
```

Comment:
I don't think this method needs to be part of this interface. I try to keep the WorldModel.h interface read only whenever possible. Since this is only ever called on CARMAWorldModel in the WMListener it doesn't need to be part of the public interface here.

Reply:
ah okay makes sense